src/Bicep.LangServer/Handlers/BicepRenameHandler.cs

```diff
@@ -42,6 +43,12 @@ public BicepRenameHandler(ISymbolResolver symbolResolver) : base()
                 return Task.FromResult<WorkspaceEdit?>(null);
             }
 
+            if (!Lexer.IsValidIdentifier(request.NewName))
+            {
+                // if the value that the user wants to rename to is invalid (contains characters, etc.), the rename will fail.
```

Comment:
>if [](http://example.com/codeflow?start=19&length=2)

Can you update the comment to explain that we didn't fail the request because there doesn't seem to be a good way to signal that in Omnisharp even if the LSP protocol technically supports it?